lib/elixir/lib/process.ex

```diff
@@ -399,8 +398,8 @@ defmodule Process do
 
   Inlined by the compiler.
   """
-  @spec demonitor(reference) :: true
   @spec demonitor(reference, options :: [:flush | :info]) :: boolean
+  @spec demonitor(reference, []) :: true
```

Comment:
this is the only change that is not a removal but an improvement to the definition